app/data_client.go

```diff
@@ -109,6 +109,37 @@ type TabularData struct {
 	TimeReceived  time.Time
 }
 
+// ExportTabularDataReturn represents the result of an ExportTabularData API call.
+type ExportTabularDataReturn struct {
+	OrganizationID   string
+	LocationID       string
+	RobotID          string
+	RobotName        string
+	PartID           string
+	PartName         string
+	ComponentName    string
+	ComponentType    string
+	MethodName       string
+	TimeCaptured     *timestamppb.Timestamp
+	MethodParameters map[string]*anypb.Any
+	Tags             []string
+	Payload          *structpb.Struct
```

Comment:
Likewise, `Payload` should be of type `map[string]interface{}`. There are other functions that also convert to this type.